curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1843,6 +1860,16 @@ CURVEFS_ERROR DataCache::Flush(uint64_t inodeId, bool force) {
                 dirty_.store(true, std::memory_order_release);
                 return CURVEFS_ERROR::INTERNAL;
             }
+            if (useDiskCache) {
```

Comment:
why put diskcache metric in here?

BTW, I think the design can be optimized, because whether the backend is object storage service or disk cache should be transparent to you.

Just abstract out an interface layer, like put/get, both disk cache and object storage service should implement these interfaces.

Reply:
This can be understood as metrics statistics for the underlying interface